### PR TITLE
spectre-meltdown-checker: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/linux/spectre-meltdown-checker.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`spectre-meltdown-checker --explain`
+
+- Don't use the `/sys` interface even if present:
+
+`spectre-meltdown-checker --no-sysfs`
+
+- Check for specific variants (`1`, `2`, `3`, `3a`, `4`, `l1tf`, `msbds`, `mfbds`, `mlpds`, `mdsum`, `taa`, `mcespc`, `srbds` - defaults to all):
+
+`spectre-meltdown-checker --variant {{3a}} --variant {{mcespc}}`
```
Comment (bl-ue) [inline pages/linux/spectre-meltdown-checker.md:20]:
```suggestion
- Check for specific variants (defaults to all):

`spectre-meltdown-checker --variant {{1|2|3|3a|4|l1tf|msbds|mfbds|mlpds|mdsum|taa|mcespc|srbds}}`
```

File: pages/linux/spectre-meltdown-checker.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# spectre-meltdown-checker
+
+> Spectre and Meltdown mitigation detection tool.
+> More information: <https://manned.org/spectre-meltdown-checker.1>.
+
+- Check the currently running kernel for Spectre and Meltdown:
+
+`spectre-meltdown-checker`
+
+- Check the currently running kernel and show explanations for taken actions to mitigate a vulnerability:
```
Comment (bl-ue) [inline pages/linux/spectre-meltdown-checker.md:10]:
```suggestion
- Check the currently running kernel and show an explanation of the actions to take in order to mitigate a vulnerability:
```

File: pages/linux/spectre-meltdown-checker.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Don't use the `/sys` interface even if present:
+
+`spectre-meltdown-checker --no-sysfs`
+
+- Check for specific variants (`1`, `2`, `3`, `3a`, `4`, `l1tf`, `msbds`, `mfbds`, `mlpds`, `mdsum`, `taa`, `mcespc`, `srbds` - defaults to all):
+
+`spectre-meltdown-checker --variant {{3a}} --variant {{mcespc}}`
+
+- Inspect a non-running kernel:
```
Comment (bl-ue) [inline pages/linux/spectre-meltdown-checker.md:22]: to be more consistent with your previous wording...
```suggestion
- Check a non-running kernel:
```

File: pages/linux/spectre-meltdown-checker.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Check the currently running kernel for Spectre and Meltdown:
+
+`spectre-meltdown-checker`
+
+- Check the currently running kernel and show explanations for taken actions to mitigate a vulnerability:
+
+`spectre-meltdown-checker --explain`
+
+- Don't use the `/sys` interface even if present:
```
Comment (bl-ue) [inline pages/linux/spectre-meltdown-checker.md:14]: How about moving this example down below the next example, i.e. to the 4th position?  the next one seems more essential.

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: pages/linux/spectre-meltdown-checker.md (hunk 4 above)
Comment (marchersimon) [reply pages/linux/spectre-meltdown-checker.md:14]: Sounds reasonable

File: pages/linux/spectre-meltdown-checker.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# spectre-meltdown-checker
+
+> Spectre and Meltdown mitigation detection tool.
+> More information: <https://manned.org/spectre-meltdown-checker.1>.
+
+- Check the currently running kernel for Spectre and Meltdown:
```
Comment (bl-ue) [inline pages/linux/spectre-meltdown-checker.md:6]:
```suggestion
- Check the currently running kernel for Spectre or Meltdown:
```
?

**Review by bl-ue — APPROVED** [review]

File: pages/linux/spectre-meltdown-checker.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# spectre-meltdown-checker
+
+> Spectre and Meltdown mitigation detection tool.
+> More information: <https://manned.org/spectre-meltdown-checker.1>.
+
+- Check the currently running kernel for Spectre or Meltdown:
+
+`spectre-meltdown-checker`
```
Comment (navarroaxel) [inline pages/linux/spectre-meltdown-checker.md:8]: Without `sudo` I get this warning:
```
Note that you should launch this script with root privileges to get accurate information.
We'll proceed but you might see permission denied errors.
To run it as root, you can try the following command: sudo /usr/bin/spectre-meltdown-checker
```
So, maybe we should add `sudo` to the commands. What do you think @bl-ue ?

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Could you add an example for formatted output:
```
spectre-meltdown-checker --batch {{text|json|nrpe|prometheus|short}}
```

File: pages/linux/spectre-meltdown-checker.md (hunk 6 above)
Comment (bl-ue) [reply pages/linux/spectre-meltdown-checker.md:8]: Yeah, make sense to me.

Comment (marchersimon) [reply pages/linux/spectre-meltdown-checker.md:8]: I completely overlooked that. It is even mentioned in the Github repository.

File: pages/linux/spectre-meltdown-checker.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Check the currently running kernel and show an explanation of the actions to take in order to mitigate a vulnerability:
+
+`sudo spectre-meltdown-checker --explain`
+
+- Check for specific variants (defaults to all):
+
+`sudo spectre-meltdown-checker --variant {{1|2|3|3a|4|l1tf|msbds|mfbds|mlpds|mdsum|taa|mcespc|srbds}}`
+
+- Print the output using a specific formatting:
```
Comment (navarroaxel) [inline pages/linux/spectre-meltdown-checker.md:18]:
```suggestion
- Display output using a specific output format:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**